tracer-core/src/test/java/com/alipay/common/tracer/core/TestUtil.java

```diff
@@ -43,9 +45,21 @@ public static boolean compareSlotMap(String a, String b) {
         return true;
     }
 
-    public static void waitForAsyncLog() throws InterruptedException {
```

Comment:
There's also SelfLogTest that depends on this method

Reply:
@glmapper Fixed